rts/Sim/MoveTypes/GroundMoveType.cpp

```diff
@@ -1385,11 +1387,16 @@ void CGroundMoveType::ChangeHeading(short newHeading) {
 	if (owner->GetTransporter() != nullptr)
 		return;
 
+	if (wantedHeading != newHeading)
+		wantedHeading = newHeading;
```

Comment:
```suggestion
	wantedHeading = newHeading;
```

Reply:
To check: does a non-changing store avoid a cache line write-back?

Reply:
Okay. Checking through the details. Intel does implement tracking of cache line state.